front/lib/vault_rollout.ts

```diff
@@ -2,7 +2,8 @@ import type { Authenticator } from "@app/lib/auth";
 import { VaultResource } from "@app/lib/resources/vault_resource";
 
 export const getDustAppsListUrl = async (
-  auth: Authenticator
+  auth: Authenticator,
+  vaultId?: string
```

Comment:
NIT: Why keeping possibility of not passing the vault? We could simplify and have it always.

Reply:
Good point as routes are not duplicated